src/jupyterlabcontroller/constants.py

```diff
@@ -29,6 +30,15 @@
 IMAGE_REFRESH_INTERVAL = timedelta(minutes=5)
 """How frequently to refresh the list of remote and cached images."""
 
+KUBERNETES_DELETE_TIMEOUT = timedelta(seconds=60)
```

Comment:
Is this long enough for namespaces?  I guess if there's a running pod and you haven't said otherwise, it's 30s, so probably so.  We can always change it if we need to, I guess.

Reply:
It's not, but namespaces will get their own timeout handling. I'm not entirely happy with this being a constant, though, and timeouts were a bit messy. I have it on my list to go back and take another pass at timeout handling later.